ethcore/src/engines/tendermint/mod.rs

```diff
@@ -37,11 +37,12 @@ use ethkey::{recover, public_to_address, Signature};
 use account_provider::AccountProvider;
 use block::*;
 use spec::CommonParams;
-use engines::{Engine, Seal, EngineError};
+use engines::{Engine, Seal, EngineError, ConstructedVerifier};
+use engines::authority_round::{combine_proofs, destructure_proofs};
```

Comment:
could the `tendermint` module get its own versions of these functions (as the engines will be split into seperate crates in the future)? also, `tendermint` proofs don't need "signal_number" because `signal_number` is always the number of the header passed to `epoch_verifier`. That means the proofs can be slightly more compact at 2 elements instead of 3.